app/code/Magento/Customer/Model/Indexer/Source.php

```diff
@@ -82,7 +82,7 @@ public function addFieldToFilter($attribute, $condition = null)
     /**
      * @inheritdoc
      */
-    public function count()
+    public function count(): int
```

Comment:
```suggestion
    #[\ReturnTypeWillChange]
    public function count()
```

Reply:
Hello @ihor-sviziev
This is really true.
This is big BIC IMHO.
We fixed this changes.
@karyna-tsymbal-atwix thank you for the fast fix.